tests/e2e/cucumber/features/smoke/search.feature

```diff
@@ -105,6 +106,21 @@ Feature: Search
       | folder   |
       | FolDer   |
 
+
+    # search difficult names
+    When "Alice" searches "strängéनेपालीName" using the global search and the "everywhere" filter and presses enter
+    Then following resources should be displayed in the files list for user "Alice"
+      | strängéनेपालीName |
+
+    # deleting folder from search result and search deleted resource
+    When "Alice" deletes the following resource using the sidebar panel
+      | resource          | from |
+      | strängéनेपालीName |      |
```

Comment:
The indent is not right here or is it because of the characters?
```suggestion
      | resource                 | from |
      | strängéनेपालीName |          |
```

Reply:
here is correct. seems wrong due characters